src/Http/Http.Results/test/EmptyResultTest.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Http.Result;
+
+public class EmptyResultTest
+{
+    public async Task EmptyResult_DoesNothing()
```

Comment:
Does this need a `[Fact]` to declare it as an actual test?

```suggestion
    [Fact]
    public async Task EmptyResult_DoesNothing()
```

Reply:
Good eye!

Reply:
Added in the latest commit. Thanks for the catch! But still, the test did execute without this attribute! I wonder!